iac/tf-multienv-cicd-anthos-autopilot/shell-scripts/04-init-cloud-sql.sh

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
```

Comment:
This script ends up replacing: https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/extras/cloudsql/create_cloudsql_instance.sh correct?

Reply:
Will there still be a non-TF-friendly way of deploying with Cloud SQL through kustomize / skaffold alone?

Reply:
> This script ends up replacing: https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/extras/cloudsql/create_cloudsql_instance.sh correct?
Not as far as I can tell. The linked script is setting up the infrastructure (CloudSQL instances). This is done entirely in terraform in this PR ([staging](https://github.com/aablsk/bank-of-anthos/blob/2b6f6f77b6e2d116bc705f696efc274515035df9/iac/tf-multienv-cicd-anthos-autopilot/env-staging.tf#L70-L99), [production](https://github.com/aablsk/bank-of-anthos/blob/2b6f6f77b6e2d116bc705f696efc274515035df9/iac/tf-multienv-cicd-anthos-autopilot/env-production.tf#L70-L98])). The purpose of `04-init-cloud-sql.sh` is to [deploy the kubernetes jobs](https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/main/extras/cloudsql/populate-jobs) that will initialise the CloudSQL instances with data.

> Will there still be a non-TF-friendly way of deploying with Cloud SQL through kustomize / skaffold alone?
There is currently no way to deploy CloudSQL through kustomize/skaffold alone. Instead what we rely on to provision infrastructure is the shell script you linked which heavily uses gcloud cli. This is still possible with the proposed solution. It could be the possibility to provision with kustomize + skaffold, but this would imply usage of [config connector](https://cloud.google.com/config-connector/docs/overview) and is out of scope for this PR.